src/test/scala/org/broadinstitute/dsde/workbench/sam/api/ResourceRoutesV2Spec.scala

```diff
@@ -107,20 +107,22 @@ class ResourceRoutesV2Spec extends FlatSpec with Matchers with TestSupport with
           .thenReturn(IO.pure(false))
     }
 
-    newParentOpt.map { newParent =>
-      when(samRoutes.resourceService.setResourceParent(mockitoEq(childResource), mockitoEq(newParent), any[SamRequestContext]))
-          .thenReturn(IO.unit)
-      actionsOnNewParent.map(action => when(samRoutes.policyEvaluatorService.hasPermission(mockitoEq(newParent), mockitoEq(action), mockitoEq(defaultUserInfo.userId), any[SamRequestContext]))
-        .thenReturn(IO(true)))
-      missingActionsOnNewParent.map(action => when(samRoutes.policyEvaluatorService.hasPermission(mockitoEq(newParent), mockitoEq(action), mockitoEq(defaultUserInfo.userId), any[SamRequestContext]))
-        .thenReturn(IO(false)))
-      if (accessToNewParent) {
-        when(samRoutes.policyEvaluatorService.listResourceAccessPoliciesForUser(mockitoEq(newParent), mockitoEq(defaultUserInfo.userId), any[SamRequestContext]))
-          .thenReturn(IO(Set(otherPolicy)))
-      } else {
-        when(samRoutes.policyEvaluatorService.listResourceAccessPoliciesForUser(mockitoEq(newParent), mockitoEq(defaultUserInfo.userId), any[SamRequestContext]))
-          .thenReturn(IO(Set[AccessPolicyWithoutMembers]()))
-      }
+    newParentOpt match {
+      case Some(newParent) =>
+        when(samRoutes.resourceService.setResourceParent(mockitoEq(childResource), mockitoEq(newParent), any[SamRequestContext]))
+            .thenReturn(IO.unit)
+        actionsOnNewParent.map(action => when(samRoutes.policyEvaluatorService.hasPermission(mockitoEq(newParent), mockitoEq(action), mockitoEq(defaultUserInfo.userId), any[SamRequestContext]))
+          .thenReturn(IO(true)))
+        missingActionsOnNewParent.map(action => when(samRoutes.policyEvaluatorService.hasPermission(mockitoEq(newParent), mockitoEq(action), mockitoEq(defaultUserInfo.userId), any[SamRequestContext]))
+          .thenReturn(IO(false)))
+        if (accessToNewParent) {
+          when(samRoutes.policyEvaluatorService.listResourceAccessPoliciesForUser(mockitoEq(newParent), mockitoEq(defaultUserInfo.userId), any[SamRequestContext]))
+            .thenReturn(IO(Set(otherPolicy)))
+        } else {
+          when(samRoutes.policyEvaluatorService.listResourceAccessPoliciesForUser(mockitoEq(newParent), mockitoEq(defaultUserInfo.userId), any[SamRequestContext]))
+            .thenReturn(IO(Set[AccessPolicyWithoutMembers]()))
+        }
+      case None =>
```

Comment:
since there is nothing actually in the None case, does this actually change anything? If not, I would undo.

Reply:
yeah that sounds good. When I was reviewing my code, I thought it was a little easier to read this way so I left it as is, but I'm not super attached to it!